packages/client/src/components/app/blocks/form/InnerFormBlock.svelte

```diff
@@ -0,0 +1,194 @@
+<script>
+  import BlockComponent from "components/BlockComponent.svelte"
+  import Placeholder from "components/app/Placeholder.svelte"
+  import { makePropSafe as safe } from "@budibase/string-templates"
+
+  export let dataSource
+  export let actionUrl
+  export let actionType
+  export let size
+  export let disabled
+  export let fields
+  export let labelPosition
+  export let title
+  export let showSaveButton
+  export let showDeleteButton
+  export let schema
+  export let repeaterId
+
+  const FieldTypeToComponentMap = {
+    string: "stringfield",
```

Comment:
Having a look at the component map, it seems this type is missing: `"field/sortable": SortableFieldSelect,`
Just wondering if that matters?

Reply:
This is a map for generating form fields from schema - its not the map for component settings which is what you're thinking of! That map is defined in `componentSettings.js`.